tests/Unit/IO/CMakeLists.txt

```diff
@@ -33,3 +33,8 @@ spectre_add_python_test(
   "Unit.IO.H5.Python"
   "Test_H5.py"
   "unit;IO;H5;python")
+spectre_add_python_test(
```

Comment:
can you add a blank line above the `spectre_add_python_test` so that the file is a bit easier to read at a quick glance?